app/src/pages/HomePage.tsx

```diff
@@ -193,12 +211,13 @@ export const HomePage = (): React.ReactElement => {
           <Stack direction={Direction.Vertical} childAlignment={Alignment.Start} contentAlignment={Alignment.Center} shouldAddGutters={true} defaultGutter={PaddingSize.Wide}isScrollableHorizontally={true} paddingBottom={PaddingSize.Wide3}>
             <Text variant='header3'>Choose your picture </Text>
             <Stack direction={Direction.Horizontal} childAlignment={Alignment.Center} contentAlignment={Alignment.Center} shouldAddGutters={true} shouldWrapItems={true}>
-              {imageData && imageData.map((image, index: number) : React.ReactElement => (
+              {ownerTokens && ownerTokens.map((ownerToken: Token, index: number) : React.ReactElement => (
```

Comment:
you should show a loading state while the ownerTokens are undefined. and if ownerTokens is null it should show an error